linkerd/app/integration/src/tests/telemetry.rs

```diff
@@ -1233,15 +1230,12 @@ mod transport {
             .await;
     }
 
-    #[test]
+    #[tokio::test]
     #[cfg(target_os = "macos")]
```

Comment:
@hawkw do you recall why these tests are macOS-specific? should they instead be marked flakey so we can at least catch build failures? Do we really depend on some macOS specific behavior here, or are these tests just broken?

Reply:
they are macos-specific due to errno labels; some syscalls return different error codes on macos than on Linux, so the errno labels are different on macOS. the tests assert that a label is added for "whatever error code the OS returns": https://github.com/linkerd/linkerd2-proxy/pull/1392/files/92538c2c2764bd22802ad3db9eac676828f070d7#diff-22fea522d6f7f7482dc4554c9d57ffb7bb433c7dc1729ef93ed13864af712cb2R1254

Reply:
as a potential alternative, we could just only test errno labels on Linux!